airflow/ui/src/pages/Dashboard/HistoricalMetrics/AssetEvent.tsx

```diff
@@ -0,0 +1,69 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Box, Text, HStack } from "@chakra-ui/react";
+import { FiDatabase } from "react-icons/fi";
+import { MdOutlineAccountTree } from "react-icons/md";
+import { Link } from "react-router-dom";
+
+import type { AssetEventResponse } from "openapi/requests/types.gen";
+import Time from "src/components/Time";
+
+export const AssetEvent = ({
+  event,
+}: {
+  readonly event: AssetEventResponse;
+}) => {
+  const hasDagRuns = event.created_dagruns.length > 0;
+  const source = event.extra?.from_rest_api === true ? "API" : "";
+
+  return (
+    <Box fontSize={13} mt={1} w="full">
+      <Text fontWeight="bold">
+        <Time datetime={event.timestamp} />
+      </Text>
+      <HStack>
+        <FiDatabase /> <Text> {event.uri ?? ""} </Text>
```

Comment:
Likely better to use the name here, maybe we some rich UI that reveals more asset information on user interaction.

Reply:
Once we pass name and group we can put more useful information here. And could have a tooltip to show more.

Reply:
I have updated the PR to use name in the UI and a tooltip on hovering over name to display group and uri.